starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Apple Pie Recipe</title>
-</head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <head>
+    <link rel="stylesheet" href="css/main.css" />
+    <title>Apple Pie Recipe</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1>
+        Apple Pies
+      </h1>
+    </header>
+  
+    <section id="intro">
+        <p>
+            This was my grandmother's apple pie recipe. I have never seen another
+            one quite like it. It will always be my favorite and has won me several
+            first place prizes in local competitions. I hope it becomes one of your
+            favorites as well!
+          </p>
+          <img src="images/cooking-info.png" alt="okay" />
+    </section>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+    <section id="ingredients" class="container">
+      <hr />
+      <h2>
+        Ingredients
+      </h2>
+      <p>
+        -1 recipe pastry for a 9 inch double crust pie <br />
+        -1/2 cup unsalted butter <br />
+        -3 tablespoons all-purpose flour <br />
+         -1/4 cup water <br>
+          -3 tablespoons all-purpose
+        flour <br>
+         -1/4 cup water <br>
+          -1/2 cup white sugar <br>
+           -1/2 cup packed brown sugar <br>
+          -8 Granny Smith apples - peeled, cored and sliced
+      </p>
+      <img src="images/recipe-info.png" alt="okaa" />
+    </section>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
-
-Place the bottom crust in your pan. Fill with apples, mounded slightly. Cover with a
-lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-that it does not run off.
-
-Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
-degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
-</body>
+    <section id="directions" class="container">
+      <hr />
+      <h2>
+        Directions
+      </h2>
+      
+        1)Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a
```

Comment:
your directions here should be in an <ol> list (the directions are always going to come in the same order)